server/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const { get } = require('./controller/checkoutController.js');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '..', 'public')));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+  extended: true,
+}));
+app.get('/api/checkout/:id', get);
+const PORT = 3010;
+app.listen(PORT);
```

Comment:
Something that I've seen as a nice to have is linking localhost here so you can quickly access it upon start... Not required.